lib/registries/addon/components/registries-metadata/template.hbs

```diff
@@ -146,11 +146,20 @@
                 @readOnly={{not this.registration.userHasAdminPermission}}
             />
         </div>
-        {{!--
         <div local-class='Field'>
             <h4>{{t 'registries.registration_metadata.citation'}}</h4>
-            {{! TODO EMB-448 }}
+            <OsfButton
+                data-analytics-name='Expand citations'
+                local-class='LinkButton'
+                @type='link'
+                @onClick={{action (mut this.expandCitations) (not this.expandCitations)}}
+            >
+                {{this.registrationDisplayUrl}}
+                <FaIcon @icon='caret-down' />
+            </OsfButton>
+            {{#if this.expandCitations}}
```

Comment:
Use `EditableField` instead? We could probably fix/repurpose it to account for this case where no permissions are needed.

Reply:
Hm, this one behaves a differently from the others; I think it might be better to let `EditableField` do one specific thing, rather than handle every expand/collapse situation.

Reply:
Okay. That makes sense.